### ISSUE TITLE
Support SAS syntax checking/linting

### ISSUE THREAD

**Issue by Enzhihpp** [sassoftware/vscode-sas-extension#84]
As a user, I'd like to have SAS syntax checking/linting features.

**Comment by scottdover** [comment]
We now have formatting in place for sas code. Does this fulfill this request, @enzhihpp?

**Comment by scnwwu** [comment]
> We now have formatting in place for sas code. Does this fulfill this request, @enzhihpp?

They're different. Linter will report error/warnings for code quality issues. Formatter makes better alignment and pretty printing.

**Comment by enzhihpp** [comment]
Yes, as @scnwwu mentioned, it's different. We hope the extension could analyze sas code statically.

**Comment by engmtcdrm** [comment]
I would like to add that if this feature would be added, that it would prevent us from submitting code to run if there is a syntax error or if it did not pass linting. The number of times I've kicked off code just to spot a simple syntax error like forgetting a semi-colon would help save time.
